lib/ui/src/components/sidebar/useExpanded.ts

```diff
@@ -82,6 +87,25 @@ export const useExpanded = ({
     setExpanded({ ids: getAncestorIds(data, selectedStoryId), value: true });
   }, [data, selectedStoryId]);
 
+  const collapseAll = useCallback(() => {
+    setExpanded({ ids: Object.keys(data), value: false });
+  }, [data]);
```

Comment:
You'll have to omit the `rootIds` from this set, because those can never be collapsed.
This should probably do the trick:

```suggestion
  const collapseAll = useCallback(() => {
    const ids = Object.keys(data).filter(id => !rootIds.includes(id));
    setExpanded({ ids, value: false });
  }, [data, rootIds]);

```

For `expandAll` this doesn't really matter, because the rootIds will already be set to `true`.